exercises/elm-1-a/README.md

```diff
@@ -0,0 +1,17 @@
+## Introduction
+
+This is Part 1 of our research into how people write Elm differently. This exercise should take 10-15 minutes. There is no right or wrong way to approach it - just do what feels the most natural to you. Feel free to use books, Google or Stack Overflow, just like you would if you were programming normally, but please don't actively look for a solution to the problem as this will negatively affect the research.
+
+## Instructions
```

Comment:
Maybe be explicit about which method the student is expected to define? See e.g. the [C# instructions](https://github.com/exercism/research_experiment_1/blob/master/exercises/csharp-1-a/README.md#instructions).